repos/system_upgrade/el7toel8/actors/storagescanner/libraries/library.py

```diff
@@ -79,22 +80,41 @@ def _get_fstab_info(fstab_path):
     ''' Collect storage info from /etc/fstab file '''
     if _is_file_readable(fstab_path):
         with open(fstab_path, 'r') as fstab:
-            for entry in fstab:
+            keys = ['fs_spec', 'fs_file', 'fs_vfstype', 'fs_mntops', 'fs_freq', 'fs_passno']
+            for line, entry in enumerate(fstab, 1):
                 if entry.startswith('#'):
                     continue
 
+                if '#' in entry:
+                    message = "Your /etc/fstab has '#' comment in line {} please verify" \
```

Comment:
Please do not use \ for string continuation rather use parenthesis. Logging format strings should usually prefer to use %s.
Strings are supposed to use single quotes not double quotes. 
```suggestion
                    api.current_logger().warning(
                        'Detected in /etc/fstab a "#" comment in line %s please verify if it is correct.', line)
```